src/features/collapsible-sidebar/CollapsibleSidebar.scss

```diff
@@ -0,0 +1,149 @@
+/* Sidebar Menu */
+@import '../../styles/sizes.media';
+@import '../../styles/variables';
+
+$collapsible-side-bar-width: 240px;
+$collapsible-side-bar-collapsed-width: 68px;
+$collapsible-side-bar-h-padding: $bdl-grid-unit * 3;
+$collapsible-side-bar-v-padding: $bdl-grid-unit * 3;
+$left-sidebar-z-index: 40 !default;
+
+.bdl-CollapsibleSidebar-wrapper {
+    position: relative;
+    flex-shrink: 0;
+    width: $collapsible-side-bar-width;
```

Comment:
This needs an additional media query I think so it does't break at small sizes. Would be good to not carry that bug into this repo.